multi_view:
  front : [[0.035383036947178384, -0.11473520506707088, -0.99276576966302177],[0.33174556112878911, -0.096479639115394977, -0.93842237926613203], [0.64792867802836396, -0.09877952620397483, -0.75526884841845232], [0.84069623781529446, -0.057016937367817905, -0.53849689374814824], [0.99861447483575372, 0.0095433475409705988, -0.051749929142214289]]
  lookat : [[0.038923033325571767, 0.11577049089809685, 0.95233340037115533],[-0.015091367119826041, 0.11337003221324685, 0.93348533942550582], [-0.082389701902882792, 0.10829077301769802, 0.88378544822574601], [-0.082389701902882792, 0.10829077301769802, 0.88378544822574601], [-0.096112757113331185, 0.092377226951892391, 0.75000254321344495]]
  up : [[0.029754411549334947, -0.99282647579126126, 0.11580269410189503],[0.030582102574529486, -0.99313381172004922, 0.11291575187070634], [0.011873286190625304, -0.99012528507193198, 0.13968158409848136], [0.01802913542122616, -0.990942704008271, 0.13306955943693297], [0.016047446312962718, -0.99180293091586658, 0.12676523850618271]]
  zoom : [0.69999999999999978, 0.69999999999999987, 0.69999999999999987, 0.69999999999999987, 0.69999999999999987]

rotation_matrix : 
  rotation_matrix_left : [[0.0000000,  0.0000000,  -1.0000000,0.885],[0.0000000,  1.0000000,  0.0000000,0.03],[1.0000000,  0.0000000,  0.0000000,0.665],[0,0,0,1]]